tests/e2e/pageobjects/ide/OpenWorkspaceWidget.ts

```diff
@@ -31,9 +31,14 @@ export class OpenWorkspaceWidget {
         const pathNodes: string[] = path.split('/');
         let buildPath: string = '';
 
-        const promises = pathNodes.map(async currentPath => {
+        const promises = pathNodes.map(async (currentPath, index) => {
             buildPath += `/${currentPath}`;
-            await  this.driverHelper.waitAndClick(By.id(buildPath), timeout);
+
+            if (index === 0) {
```

Comment:
IMHO it would be helpful to have comment in the code about which particular problem this workaround resolves.

Reply:
But, I don't think that it's a workaround. Now there is such behavior with locators in the 'OpenWorkspaceWidget', see screenshot. The first item (`index=0` -> `/projects`) has locator which can be found 'By.id', but next (`/projects/node-js`) requires another - 'By.xpath'. My guess this change could be related to switch on by default 'multi-root', but it is only guessing.

Reply:
Got it.
Your answer could be used as a comment in the code:
> The first item (index=0 -> /projects) has locator which can be found 'By.id', but next (/projects/node-js) requires another - 'By.xpath'.

Reply:
Ok, I'll add the comment

Reply:
thanks